cmd/res/profiles/modbus.test.device.string.profile.yml

```diff
@@ -0,0 +1,41 @@
+name: "Test.Device.Modbus.String.Profile"
+manufacturer: "no manufacturer"
+model: "string value type"
+labels:
+  - "no lable"
+description: "this profile is testing  string value type with read and write"
+
+deviceResources:
+  -
+    name: "StringA"
+    description: "no description"
+    attributes:
+      { primaryTable: "INPUT_REGISTERS", startingAddress: "1", stringRegisterSize: "1"}
```

Comment:
The startingAddress and stringRegisterSize should be integer. Please change to `{ primaryTable: "INPUT_REGISTERS", startingAddress: 1, stringRegisterSize: 1}`